src/Dashboard/Main.tsx

```diff
@@ -42,8 +35,6 @@ class DashboardContainer extends React.Component<{}, IDashboardState> {
       status: HackerStatus.HACKER_STATUS_NONE,
       confirmed: true,
     };
-    this.confirmAccountToastError = this.confirmAccountToastError.bind(this);
```

Comment:
Unless I’m mistaken, I believe we still need to bind these two methods, because we’re passing them down into other components.

Reply:
`confirmAccountToastError` is now a public field and doesn't need to have a binded `this` anymore.

See LOC: https://github.com/hackmcgill/hackerAPI-frontend/pull/243/files/f8f5da2bf2bf8f9dc6989c87e9debd0d0a3934dc#diff-96eda7d06259adf9ee7131dbdf9635f6R95

This is an experimental feature available via Babel that comes bundled with CRA 2.0. https://babeljs.io/docs/en/babel-plugin-proposal-class-properties

Motivation: https://codeburst.io/use-class-properties-to-clean-up-your-classes-and-react-components-93185879f688

I think it'd be good to convert this project-wide?